### PR TITLE
Timing core clk bugfix at init

### PR DESCRIPTION
**Pull request by dawoodalnajjar**
When initiating LCLSII timing configuation sequence, sometimes the Rx enters an unknown state, and the RxClk goes to zero. Adding delays in the configuration sequence resolves this issue. t

### CODE REVIEW

File: python/epix_hr_leap_common/_TimingRx.py [hunk 1]
```diff
@@ -58,16 +58,24 @@ def __init__( self,sim=False,**kwargs):
...
+            time.sleep(0.1)
             self.TxDbgPhyRst.set(0x1)
+            time.sleep(0.1)
             self.TxDbgPhyRst.set(0x0)
+            time.sleep(0.1)
             self.TimingFrameRx.ModeSelEn.setDisp('UseClkSel')
             self.TimingFrameRx.RxPllReset.set(1)
             time.sleep(1.0)
             self.TimingFrameRx.RxPllReset.set(0)
+            time.sleep(0.1)
```
Comment (jumdz) [inline python/epix_hr_leap_common/_TimingRx.py:70]: Do all these sleeps are needed or was it for debugging ?

File: python/epix_hr_leap_common/_TimingRx.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def __init__( self,sim=False,**kwargs):
 
         self.add(l2si.TriggerEventManager(
             offset       = 0x0004_0000,
-            numDetectors = 1,
+            numDetectors = 2,
```
Comment (jumdz) [inline python/epix_hr_leap_common/_TimingRx.py:51]: Is it compatible with the other setups ?

**Review by jumdz — APPROVED** [review]

File: python/epix_hr_leap_common/_TimingRx.py (hunk 1 above)
Comment (dawoodalnajjar) [reply python/epix_hr_leap_common/_TimingRx.py:70]: Some sleeps were there in the original code. The sleeps added are 0.1 seconds. Maybe not all of them are needed, but this sequence seems to "unhang" the IP

File: python/epix_hr_leap_common/_TimingRx.py (hunk 2 above)
Comment (dawoodalnajjar) [reply python/epix_hr_leap_common/_TimingRx.py:51]: With my and your setup, yes

**Pull request merged**